chap-intro.tex

```diff
@@ -645,26 +562,14 @@ \subsection{CHERI-RISC-V, CHERI-x86-64, and Arm Morello}
 
 Even when targeting a specific architecture, there are multiple ways to
 express CHERI's higher-level memory-protection and security-model goals.
-For example, the CHERI model can be implemented using a \textit{split
```

Comment:
I think we could keep this sentence, but I also don't mind deleting it.

Reply:
In general I'm removing almost all references to split register files except for a historical note in the rationale chapter.  (I have a followup branch that finishes removing it from CHERI-RISC-V which this PR doesn't yet fully do.)  In that vein I think it makes sense to remove it here.